### PR TITLE
Address some translation comments in PR #4359

### PR DESCRIPTION
**Pull request by xingzhou**
Addressed some translation comments in PR #4359

@tengqm and @chenopis, PTAL, thanks

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4579)

### CODE REVIEW

**Comment by chenopis** [comment]
@markthink Can you take a look and approve this? Thx

File: cn/docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 1]
```diff
@@ -199,13 +199,13 @@ subnet 10.20.30.0 netmask 255.255.255.0 {
...
+在本示例中，我们将通过静态方式创建一个etcd服务器，用于运行Kubernetes主控组件，并用作etcd主节点。
 
-由于我们的集群处于一个离线的环境中，所以大部分的CoreOS和Kubernetes帮助进程是受限的。为了完成部署，我们需要下载Kubernetes的各种可执行文件到本地然后再启动运行。
+由于我们的集群处于一个离线的环境中，所以大部分的CoreOS和Kubernetes帮助进程是受限的。为了完成部署，我们需要下载Kubernetes的各个可执行文件到本地然后再启动运行。
 
-一种简单的方案是在DHCP/TFTP主机上搭建一个简易的web服务器，从而环境中的CoreOS PXE机器可以从其上下载各种可执行文件。
+一种简单的方案是在DHCP/TFTP主机上搭建一个简易的web服务器，从而环境中的CoreOS PXE机器可以从其上下载各个可执行文件。
 
-为了达到这一目标，我们将启动一个`apache`服务器并提供运行Kuberetes所需要的各种可执行文件。
+为了达到这一目标，我们将启动一个`apache`服务器并提供运行Kuberetes所需要的各个可执行文件。
```
Comment (tengqm) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: please fix the typo. other than that, looks good to me.

Comment (xingzhou) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: Done

**Pull request merged**